frontend/src/app/features/map/siteDrawer/SiteDetailsDrawer.tsx

```diff
@@ -65,42 +89,79 @@ export const SiteDetailsDrawer: FC = () => {
       }}
       title="Selected Site"
     >
-      <div className="mb-2">
-        <span className="fw-bold mb-2">Site ID:</span> {selectedSiteId}
-      </div>
-      {loading && <SpinnerIcon size={20} className="site-fa-spin" />}
-      {!loading && (
-        <div className="d-grid gap-3">
-          <h4 className="fw-bold">{formatAddress(siteData)}</h4>
-          <div className="site-drawer-info-summary">
-            <SummaryItem
-              label="Latitude"
-              value={formatCoordinates([
-                siteData?.latDegrees,
-                siteData?.latMinutes,
-                siteData?.latSeconds,
-              ])}
+      <div className="d-flex flex-column gap-3">
+        <div className="d-flex gap-2 flex-column flex-sm-row justify-content-between">
+          <Button
+            onClick={zoomToSite}
+            variant="secondary"
+            className="justify-content-center"
+            disabled={siteDetailsLoading}
+          >
+            <MagnifyingGlassPlusIcon />
+            Zoom To
+          </Button>
+          <div className="d-flex justify-content-end gap-2 flex-column flex-sm-row">
+            <Link
+              to={`/site/details/${selectedSiteId}`}
+              className="justify-content-center"
+            >
+              <Button
+                variant="secondary"
+                className="justify-content-center w-100"
+              >
+                View Site Details
+              </Button>
+            </Link>
+            <AddToFolio
+              selectedSiteIds={[selectedSiteId || '']}
+              label="Add to Folio"
+              triggerClassName="justify-content-center"
             />
-            <SummaryItem
-              label="Site Risk Classification"
-              value={siteData?.siteRiskCode || 'N/A'}
-            />
-            <SummaryItem
-              label="Longitude"
-              value={formatCoordinates([
-                siteData?.longDegrees,
-                siteData?.longMinutes,
-                siteData?.longSeconds,
-              ])}
-            />
-            <SummaryItem label="Region" value={siteData?.city} />
-          </div>
-          <div>
-            <div className="fw-bold mb-2">Location Description</div>
-            <div>{siteData?.generalDescription}</div>
+            <Button className="justify-content-center">
+              <ShoppingCartIcon />
+              Add to Cart
+            </Button>
```

Comment:
Please note that the Add to Cart button doesn't do anything yet. I learned that the `addCartItem` mutation requires a non-nullable `userId` variable to be passed, instead of getting the current user data from the request object. I'd like to refactor it (and other cart-related mutations and queries) before adding it here.